server/services/supportedPrisonsService.ts

```diff
@@ -14,6 +14,10 @@ export default class SupportedPrisonsService {
 
   private lastUpdated = 0
 
+  async getActiveAgencies(): Promise<string[]> {
+    return this.getSupportedPrisonIds(undefined)
```

Comment:
Calls to `/info` are unauthenticated so there is no `username`. So, call the API without a username which means a system client token gets generated and cached as `%ANONYMOUS%`.